system/index.js

```diff
@@ -46,9 +52,10 @@ module.exports = async (message) => {
   const headers = message.headers.toRiffHeaders();
   Object.keys(headers).map((key) => { headers[key] = message.headers.getValue(key) });
 
-  const logLevel = DEBUG ? SF.LoggerLevel.DEBUG : SF.LoggerLevel.INFO;
   const requestId = headers['ce-id'] || headers['x-request-id'];
-  const logger = await createLogger(logLevel, requestId);
+  if (requestId) {
+    logger.addField('request_id', requestId);
```

Comment:
actually, this won't work.   I'll be adding a requestId for each new request that comes in.